rllib/tests/git_bisect/debug_learning_failure_git_bisect.py

```diff
@@ -183,7 +183,7 @@
         subprocess.run(
             "sudo apt-get install -y build-essential curl unzip psmisc".split(" ")
         )
-        subprocess.run("pip install cython==0.29.26 pytest".split(" "))
+        subprocess.run("pip install cython==0.29.37 pytest".split(" "))
```

Comment:
@sven1977 , do the rllib people still use this bisect tool?

fwiw, we have repo wide bisect tool now.